point_pattern.py

```diff
@@ -58,6 +60,12 @@ def find_largest_city(gj):
     """
     city = None
     max_population = 0
+    for place in gj["features"]:
+        thisCity = place["properties"]["pop_max"]
+        if thisCity > max_population:
+            max_population = thisCity
+            city = place["properties"]
+
```

Comment:
This logic looks good.  I suspect the issue has to do with reading the geojson file correctly.